apps/smart-contracts/core/test/utils.ts

```diff
@@ -9,6 +9,7 @@ export const FEE_DENOMINATOR = 1000000
 export const FEE_LIMIT = 50000
 export const MAX_PRICE = parseEther('1')
 export const DEFAULT_TIME_DELAY = 5
+export const RESERVE_DENOMINATOR = 1000000
```

Comment:
```suggestion
export const PERCENT_DENOMINATOR = 1000000
```
and can use this instead of FEE_DENOMINATOR above